lib/api/controllers/bulkController.js

```diff
@@ -57,6 +58,11 @@ class BulkController extends BaseController {
     assertHasBody(request);
     assertBodyHasAttribute(request, 'bulkData');
 
+    // @todo pass directly body.bulkData
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)